ocs_ci/ocs/ui/base_ui.py

```diff
@@ -78,44 +78,67 @@ def __init__(self, driver: WebDriver):
             Path(self.dom_folder).mkdir(parents=True, exist_ok=True)
         logger.info(f"screenshots pictures:{self.dom_folder}")
 
-    def do_click(self, locator, timeout=30, enable_screenshot=False, copy_dom=False):
+    def do_click(
+        self,
+        locator,
+        timeout=30,
+        enable_screenshot=False,
+        copy_dom=False,
+        avoid_stale=False,
+    ):
         """
         Click on Button/link on OpenShift Console
 
         locator (set): (GUI element needs to operate on (str), type (By))
         timeout (int): Looks for a web element repeatedly until timeout (sec) happens.
         enable_screenshot (bool): take screenshot
         copy_dom (bool): copy page source of the webpage
+        avoid_stale (bool): if got StaleElementReferenceException, caused by reference to stale, cached element,
+        refresh the page once and try click again
+        * don't use when refreshed page expected to be different from initial page, or loose input values
         """
-        if (
-            version.get_semantic_version(get_ocp_version(), True)
-            <= version.VERSION_4_11
-        ):
-            try:
-                wait = WebDriverWait(self.driver, timeout)
+
+        def _do_click(_locator, _timeout=30, _enable_screenshot=False, _copy_dom=False):
```

Comment:
Why referencing them as private?

Reply:
this is internal function unavailable for use. To make this block work without changing name of the function I had to rename initial func.
"""
The underscore prefix is meant as a hint to another programmer that a variable or method starting with a single underscore is intended for internal use
"""